flepimop/gempyor_pkg/tests/outcomes/test_run_parallel_outcomes.py

```diff
@@ -0,0 +1,74 @@
+import multiprocessing as mp
+import os
+from pathlib import Path
+
+import pytest
+
+from gempyor.testing import run_test_in_separate_process, setup_example_from_tutorials
+from gempyor.utils import read_directory
+
+
+@pytest.mark.skipif(
+    os.getenv("FLEPI_PATH") is None,
+    reason="The $FLEPI_PATH environment variable is not set.",
+)
+@pytest.mark.parametrize("n_jobs", (1, 2))
+@pytest.mark.parametrize("start_method", mp.get_all_start_methods())
+def test_run_parallel_outcomes_by_multiprocessing_start_method(
+    tmp_path: Path,
+    monkeypatch: pytest.MonkeyPatch,
+    n_jobs: int,
+    start_method: str,
+) -> None:
+    """
+    Test the parallelization of `run_parallel_outcomes` by multiprocessing start method.
+
+    This test:
+
+    1. Sets up the test environment by copying the necessary files to a temporary directory.
+    2. Runs a pared down version of `gempyor.simulate.simulate` in a new process.
+    3. Reads the contents of the 'hpar' directory as a DataFrame.
+    4. Tests the contents of the 'hpar' DataFrame.
```

Comment:
this could use more explanation as to what is being tested / why - this is about ensuring that each subpop gets ...unique values? the same values?